app/src/main/java/org/gammf/collabora_android/utils/AuthenticationUtils.java

```diff
@@ -3,7 +3,7 @@
 
 public class AuthenticationUtils {
 
-    private static final String SERVERIP = "http://192.168.1.111";
```

Comment:
Sorry, when i merged your online develop branch in my local repo, I forgot to not commit my local change in this file. If you want I can revert this to your online copy :blush: